helm-charts/seldon-core-operator/templates/webhook.yaml

```diff
@@ -43,38 +29,6 @@ webhooks:
       path: /validate-machinelearning-seldon-io-v1-seldondeployment
   failurePolicy: Fail
   name: v1.vseldondeployment.kb.io
-{{- if semverCompare ">=1.15.0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
:question: I'm not sure how/why this no longer exists -- was I running the wrong Make commands or missing a step?

Reply:
What is the version of your controller gen? That often tends to cause variations in the generated tokens

Reply:
It's 0.7.0 -- I had to uninstall v0.4 and use the `make` target to install v0.7, as it created a number of small differences.  The generation of this file was purely with v0.7.

Reply:
Ok that does sound like should be ok - could you try again using `make -C operator/helm create`? That should use the docker image to create it so it should be the exact same output

Reply:
That's what I did before - it's documented in [this comment, near the top](https://github.com/SeldonIO/seldon-core/pull/4235#issuecomment-1199102945).  I did notice it's using Docker to run some of the other commands in the Makefile